spec/init_factory_bot.rb

```diff
@@ -12,6 +12,10 @@
 
 Rys::PluginsManagement.all(systemic: true) do |plugin|
   FactoryBot.definition_file_paths << plugin.root.join('spec/factories').to_s
+
+  Array.wrap(plugin.paths['additional/spec/factories']).each do |path|
```

Comment:
1. why additional ?  => there is no documentation or best-practice for this "engine root" directory

1. What did you expect? "path" is absolute file system path. If you join it with Rails.root it create non-sense